### ISSUE TITLE
The package is being compiled, please try again in 3-5 minutes

### ISSUE THREAD

**Issue by rtsisyk** [cianru/huawei-publish-gradle-plugin#32]
```
> Task :publishHuaweiAppGalleryHuaweiRelease
Huawei AppGallery Publishing API: Generate Config
Huawei AppGallery Publishing API: Found build file: `OrganicMaps-21090106-huawei-release.aab`
Huawei AppGallery Publishing API: Get Access Token
Huawei AppGallery Publishing API: Get App ID
Huawei AppGallery Publishing API: Get Upload Url
Huawei AppGallery Publishing API: Upload build file '/home/runner/work/organicmaps/organicmaps/android/build/outputs/bundle/huaweiRelease/OrganicMaps-21090106-huawei-release.aab'
Huawei AppGallery Publishing API: Update App File Info
Huawei AppGallery Publishing API: Submit Review


FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':publishHuaweiAppGalleryHuaweiRelease'.
> SubmitResponse(ret=Ret(code=204144727, msg=[AppGalleryConnectPublishService]The package is being compiled, please try again in 3-5 minutes))
```

Plugin: 1.3.0 (latest)
Gradle: 7.0.2
ADP: 7.0.1

**Comment by vlad-roid** [comment]
I have the same issue, had posted it here: https://github.com/cianru/huawei-publish-gradle-plugin/issues/24
But I forgot to reply to @cosic so he closed it 😄 
It was the case for me with plugin version 1.2.4, also tried with 1.2.5 and still the same.
Gradle tried 7.0 and 7.2, same.
AGP 4.2.2.

**Comment by cosic** [comment]
@rtsisyk Hi! Have you tried using the `1.3.1-SNAPSHOT` version?

**Comment by cosic** [comment]
Fixed at [v1.3.1](https://github.com/cianru/huawei-publish-gradle-plugin/releases/tag/v1.3.1)
